techniques/system/common/1.0/failsafe.st

```diff
@@ -27,7 +27,7 @@ body common control
         inputs             => { "common/1.0/update.cf", "common/1.0/rudder-stdlib-core.cf" };
         output_prefix      => "rudder";
 
-        protocol_version   => "classic";
+        protocol_version   => "2";
```

Comment:
This won't work on older servers. We need a transition method.
- easy path: leave classic until old server are not supported anymore
- bad path: ask the user
- hard path: test both cases starting with the secure one allowing an override to forbid the old one